integration/inspektor-gadget/audit_seccomp_test.go

```diff
@@ -25,8 +25,6 @@ import (
 
 func TestAuditSeccomp(t *testing.T) {
```

Comment:
Just wondering if we should also have a `TestAuditSeccompSPO` which allows testing with SPO and is guarded by:

```
if *doNotDeploySPO {
  t.Skip("Skipping test since SPO is not deployed")
}
```

Not sure if it makes sense or not.

Reply:
I don't think it's necessary as using this manual sp installation approach fully tests the audit-seccomp gadget.